libs/ast_generic/AST_generic.ml

```diff
@@ -329,6 +329,74 @@ and canonical_name = string list
 and alternate_name = string list
 [@@deriving show { with_path = false }, eq, hash]
 
+(* Used as a parent class for the autogenerated iter visitor *)
```

Comment:
Maybe reference the fact that the visitor is generated at the very very end of this file, because of the big and long set of recursive types.